tests/01-vms-cpu-load.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: pool.kubevirt.io/v1alpha1
+kind: VirtualMachinePool
+metadata:
+  name: cpu-load
+spec:
+  replicas: 5
+  selector:
+    matchLabels:
+      kubevirt.io/vmpool: cpu-load
+  virtualMachineTemplate:
+    metadata:
+      creationTimestamp: null
+      labels:
+        kubevirt.io/vmpool: cpu-load
+    spec:
+      runStrategy: Always
+      template:
+        metadata:
+          creationTimestamp: null
+          labels:
+            kubevirt.io/vmpool: cpu-load
+        spec:
+          domain:
```

Comment:
To generate a relevant amount of CPU pressure on a node, we probably have to require more CPU cores here (`xz -T 0` is already multi-thread): e.g. even if 5 fully loaded VMs with 1 core each land of the same physical node with 8 physical cores we are still not generating enough CPU load to stress it.

Reply:
This got fixed by manuall setting guest cores and cpu requests.